Applications/Hermes/hermesflaskexample_rx.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import base64
+import cPickle
+import time
+import ConfigParser
+import os
+
+# Open configuration INI
+config = ConfigParser.RawConfigParser()
+filename = os.path.abspath("hermes.ini")
+config.read(filename)
+
+localcallsign = config.get('UNIT1', 'CALLSIGN')
+localnodeid = int(config.get('UNIT1', 'NODEID'))
+
+
+def getrxqueuesize(callsign, nodeid):
```

Comment:
Two items:

- I don't agree with using cpickle when JSON is already being used.
- This seems appropriate for use in the hermes module, not defined in the application sample file?